src/config.rs

```diff
@@ -384,6 +405,19 @@ fn parse_color_string(color_string: &str) -> Option<ansi_term::Color> {
     predefined_color
 }
 
+fn get_palette<'a>(
+    palettes: &'a HashMap<String, Palette>,
+    palette_name: &str,
+) -> Option<&'a Palette> {
+    let palette = palettes.get(palette_name);
+    if palette.is_some() {
+        log::trace!("Found color palette: {}", palette_name);
+    } else {
+        log::debug!("Could not find color palette: {}", palette_name);
```

Comment:
Not a blocker but maybe it would be better to log this as a warning to let the user know that they have something wrong with their configuration?